docs/features/kubernetes/configuration.md

```diff
@@ -188,8 +195,13 @@ regions.
 
 ##### `skipTLSVerify`
 
-This determines whether or not the Kubernetes client verifies the TLS
-certificate presented by the API server. Defaults to `false`.
+This determines whether the Kubernetes client verifies the TLS certificate
+presented by the API server. Defaults to `false`.
+
+##### `skipMetricsLookup`
+
+This determines whether the Kubernetes client looks up resource metrics
+CPU/Memory for pods returned by the API server. Defaults to `false`.
```

Comment:
Curious - as an end user, why would I ever want this off?

This notes it defaults to false.  Is there a side effect, like it reduces performance, or some version of K8s doesn't support it or something?  The experience in a UI to always display "unknown" as a user with this default seems a bit odd, without some clarifying context maybe?  That way the user will known what that trade off is to be able to decide upon it. I assume there's a good reason and I just may not know it, so hope that makes sense :)

Reply:
and i wonder also, if there's further nuance in the backend plugin about installing the custom metrics role and such, maybe it points to that or something.

Reply:
Local Kubernetes setups like kind etc don't start with the Kubernetes metrics server installed and therefore we can't get the metrics data. So in cases like this it would make sense to turn it off.

I think there is some room to improve the experience in regards to error messaging, I'm just swamped atm and will do that as a further PR when I get a chance.

Reply:
Yeah agree the DX isn't flawless but feels like an alright tradeoff for now